javascript-data-model/order-history.js

```diff
@@ -0,0 +1,44 @@
+var orderHistory = [
+  {
+    item: 'Javascript for impatient programmers',
+    orderDate: 'Aug 4, 2020',
+    orderNumber: '114-3941689-8772232',
+    itemPrice: 31.55,
+    returnWindow: 'Sep 7, 2020',
+    deliverDate: 'Aug 8, 2020'
+  },
+  {
+    item: 'The Timeless Way of Building',
+    orderDate: 'July 19, 2020',
+    orderNumber: '113-9984268-1280257',
+    itemPrice: 41.33,
+    returnWindow: 'Aug 19, 2020',
+    deliverDate: 'Jul 20, 2020'
+  },
+  {
+    item: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+    orderDate: 'July 4, 2020',
+    orderNumber: '114-2875557-9059409',
+    itemPrice: 15.98,
+    returnWindow: 'Aug 5, 2020',
+    deliverDate: 'July 7, 2020'
+  },
```

Comment:
So each of object (order) can have property called `items` and that property may hold array of item or items.